core/core/validation.py

```diff
@@ -0,0 +1,48 @@
+import re
```

Comment:
Just renamed the file since there is only one class `Validator`, so we should not call it `validators`